src/appleseed/renderer/kernel/rendering/final/adaptivepixelrenderer.cpp

```diff
@@ -139,16 +139,19 @@ namespace
                     frame.aov_images().size(),
                     frame.get_filter()));
 
-            if (m_params.m_diagnostics)
-                m_diagnostics.reset(new Tile(tile.get_width(), tile.get_height(), 2, PixelFormatFloat));
+            if (m_base_params.m_diagnostics)
```

Comment:
Wrap the body of multiline `if` statements with braces:

    if (m_base_params.m_diagnostics)
    {
        ...
    }